### ISSUE TITLE
Ellipsis class does not play nicely with card-title slots

### ISSUE THREAD

**Issue by MartinMuzatko** [quasarframework/quasar#1237]
### Software version

Quasar: 0.14.7
OS: Windows 10
Node: v8.7.0
NPM: 5.5.1
Browsers: Chrome

### What did you get as the error?

We are using Quasar to display devices as card. We make use of `q-card-title` with all slots (subtitle and right)
When I put class `ellipsis` on the title, the right slot is pushed away.
![image](https://user-images.githubusercontent.com/2950505/33601995-454b62f0-d9ae-11e7-99d6-9bb5fd9dd32d.png)

### What were you expecting?
Truncate the title at that point, where the right slot starts. I am not sure if this is possible with CSS alone.

### What steps did you take, to get the error?
This is the code:
```html
<q-card-title>
    <span class="ellipsis">{{device.ProductID}} - {{name}}</span>
    <span slot="subtitle">{{device.Description}} {{device.DeviceID}}</span>
    <div slot="right" class="row">
        <q-chip small color="primary">
            {{device.Subinterface}}
        </q-chip>
        <q-btn @click="showDevice" icon-right="settings" flat color="primary"  />
        <q-icon name="stop" :color="device.ConnectionState ? 'green' : 'red'">
            <q-tooltip>
                {{device.ConnectionState ? 'Device connected' : 'Disconnected'}}
            </q-tooltip>
        </q-icon>
    </div>
</q-card-title>
```

**Comment by delucasvb** [comment]
This seems to be an issue again? When placing a long title in the Cafe Basilico example, you can see it overflow. See this fiddle, based on the example in quasar-play:
https://jsfiddle.net/s3ek17La/4/

**Comment by rstoenescu** [comment]
@delucasvb This does not happen with v1. We're currently focusing on v1 as it is close its RC release. Will check and push a fix for v0.17 once v1 is out of the way.
